configure

```diff
@@ -3416,6 +3416,9 @@ if test -n "$_host"; then
 			_ar="$_host-ar cr"
 			_ranlib=$_host-ranlib
 			_strip=$_host-strip
+			if test -z "$SDL_NET_LIBS"; then
```

Comment:
But it is not needed on msys2 mingw, only on cross build (don't know why).

Reply:
I would say that's because these flags are only needed when building SDL2_net statically.
MSys2 mingw seems to provide a DLL version. In this case that's the DLL which links against ws2_32 and iphlpapi.
That's also a drawback of this patch, we are adding dependencies for cases where it's not needed.

Patching MXE is the proper way to fix this.

Reply:
Ok